README.md

```diff
@@ -74,13 +74,13 @@ same between the two libraries.
 When using the theme in conjunction with the [OpenEuropa Corporate Blocks](https://github.com/openeuropa/oe_corporate_blocks)
 component changing the component library will show a different footer block, namely:
 
-- The European Commission footer, shipping with a set of links and references that must be present on all European Commission sites. 
-- The European Union footer, shipping with a set of links and references that must be present on all European Union sites. 
```

Comment:
The empty spaces at the end of the lines are to enforce a line break in markdown. We need to restore them.